package.json

```diff
@@ -39,70 +39,70 @@
     "postversion": "git push && git push --tags && git checkout live && git merge develop --ff && git push && git checkout -"
   },
   "dependencies": {
-    "@primer/octicons-react": "^10.0.0",
+    "@primer/octicons-react": "^10.1.0",
     "@reach/dialog": "^0.10.5",
     "@reach/menu-button": "^0.9.1",
     "@testing-library/dom": "^7.28.1",
-    "@testing-library/user-event": "^12.1.2",
+    "@testing-library/user-event": "^12.5.0",
     "codemirror": "5.54.0",
-    "crx-bridge": "^2.1.0",
+    "crx-bridge": "^3.0.1",
```

Comment:
I'm not sure if it's a good thing to add this major bump here, as crx-bridge has changed quite a bit to add support for Firefox.

Should this one be reverted?

Reply:
I had tested it worked. But I didn't any release notes. So should I revert it?

Reply:
Did you test the build & functioning of the chrome extension? That's where `crx-bridge` is being used.

If it works, we can merge it.